model-optimizer/mo/utils/cli_parser.py

```diff
@@ -254,6 +253,10 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'and biases are quantized to FP16.',
                               choices=["FP16", "FP32", "half", "float"],
                               default='float')
+    common_group.add_argument('--transform',
+                              help='Apply additional transformations. '
+                                   'Example: --transform LowLatency[num_iterations=1].',
```

Comment:
Provide a generic description and format of a new command line parameter.

Reply:
done.